playwright-home-task-1/test-2-add-product-to-the-cart.spec.js

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from '@playwright/test';
+
+test('Verify Shopping Cart icon contains the number of products added', async ({ page }) => {
+  await page.goto('https://www.saucedemo.com/');
+  await page.locator('[data-test="username"]').fill('standard_user');
+  await page.locator('[data-test="password"]').fill('secret_sauce');
+  await page.locator('[data-test="login-button"]').click();
+  const firstProduct = await page.locator('.inventory_list > *:first-child');
+  await firstProduct.locator('[data-test="add-to-cart-sauce-labs-backpack"]').click();
+  const shoppingCartBadge = await page.locator('span.shopping_cart_badge');
+  await expect(shoppingCartBadge).toHaveText('1');
+});
+
+test('Verify the added product is displayed in the Shopping Cart', async ({ page }) => {
+  await page.goto('https://www.saucedemo.com/');
```

Comment:
Let's move his code to the `beforeEach` hook https://playwright.dev/docs/test-parameterize#before-and-after-hooks

```
  await page.goto('https://www.saucedemo.com/');
  await page.locator('[data-test="username"]').fill('standard_user');
  await page.locator('[data-test="password"]').fill('secret_sauce');
  await page.locator('[data-test="login-button"]').click();
```